### DISCUSSION TITLE
Install forum application in self-hosted docker environment

### DISCUSSION THREAD

**Discussion by Arkronus** [exoplatform/exo-community#51] [Q&A]
Hi!

I saw this post (https://www.exoplatform.com/blog/how-to-use-forums-in-exo-platform/) about forum capabilities, but I can't find this app in self-hosted EXP instance. 

How do I install it?

**Comment by rdenarie** [comment]
Hello @Arkronus,
The forum feature is deprecated and no more available in eXo Platform.
The blog article is and old one.
Thanks

**Comment by jonasbrant** [comment]
@rdenarie we are missing the forum. it was very usefull for FAQ and spaces that does some kind of support. 
We hope you decide to update it

**Reply by rdenarie** [reply]
We also user eXo as internal tool and in particular to realize support with our customers.
For this support use case we switch from forum to process and task applications which completly suits for this particular case

**Reply by Arkronus** [reply]
@rdenarie can you elaborate what "process application" do you mean?

**Comment by jonasbrant** [comment]
We are using process, has been great. 
the forum vantage is to use for FAQ. We can do it in Notes, but the students have not been considering so intuitive as a forum.

**Reply by rdenarie** [reply]
You have a video presentation here : https://www.exoplatform.com/fr/webinaire-exo-6-3-process-collaboration/
